calico-enterprise/operations/monitor/metrics/recommended-metrics.mdx

```diff
@@ -0,0 +1,574 @@
+
+### Max snapshot size (raw)
+
+| Max snapshot size (raw)          |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | <code>max(typha_snapshot_raw_bytes)</code>                              |
+| Example value                    | <code>{} 557359</code>                                                  |
+| Explanation                      | The raw size in bytes of snapshots sent from Typha to clients. |
+| Threshold value recommendation   | N/A                                                          |
+| Threshold breach symptoms        | N/A                                                          |
+| Threshold breach recommendations | N/A                                                          |
+| Priority Level                   | Optional.                                                    |
+
+### Max snapshot size (compressed)
+
+| Max snapshot size (compressed)   |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | <code>max(typha_snapshot_compressed_bytes)</code>                       |
+| Example value                    | <code>{}134845</code>                                                   |
+| Explanation                      | The compressed size in bytes of snapshots sent from Typha to clients. |
+| Threshold value recommendation   | This metric can be helpful for customers to estimate the bandwidth requirements for Felix to startup. For example, if the compressed snapshot size is 20MB in size on average, and 1000 Felix/calico-nodes start up, the bandwidth requirements could be estimated at 20GB between the pool of Typha and the set of Felixes across the network. |
+| Threshold breach symptoms        | N/A                                                          |
+| Threshold breach recommendations | N/A                                                          |
+| Priority Level                   | Optional.                                                    |
+
+## Felix cluster-state metrics
+
+### CPU usage
+
+| CPU usage                        |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | <code>rate(process_cpu_seconds_total{30s}) \* 100</code>     |
+| Example value                    | <code>{**endpoint**="metrics-port", instance="10.0.1.20:9091", **job**="felix-metrics-svc", namespace="calico-system", **pod**="calico-node-qzpkt", **service=**"felix-metrics-svc"}3.1197504199664072</code> |
+| Explanation                      | CPU in use by calico-node represented as a percentage of a core. |
```

Comment:
```suggestion
| Explanation                      | CPU in use by Felix represented as a percentage of a core. |
```

Reply:
calico node has a lot of components inside that also use memory, this metrics only counts
 Felix